GCSViews/FlightData.Designer.cs

```diff
@@ -77,6 +77,8 @@ private void InitializeComponent()
             this.myButton1 = new MissionPlanner.Controls.MyButton();
             this.myButton2 = new MissionPlanner.Controls.MyButton();
             this.myButton3 = new MissionPlanner.Controls.MyButton();
+            this.tabDroneID = new System.Windows.Forms.TabPage();

```

Comment:
given i know the plugin adds this, i dont think this should be here

Reply:
Correct. I rebased the FlightData files back to master, but looks like I missed FD Designer.

Reply:
Should be removed from PR